src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -387,9 +387,7 @@ final void printCacheStats() {
     protected static FeatureReader<VariantContext> getGenomicsDBFeatureReader(final GATKPath path, final File reference, final GenomicsDBOptions genomicsDBOptions) {
         final String workspace = IOUtils.getGenomicsDBAbsolutePath(path) ;
         if (workspace == null) {
-            throw new IllegalArgumentException("Trying to create a GenomicsDBReader from  non-GenomicsDB inputpath " + path);
-        } else if (Files.notExists(IOUtils.getPath(workspace))) {
```

Comment:
I wasn't actually suggesting that you remove this check completely, as I think it's a necessary check -- just that it might be better to normalize the workspace name to end with a slash at an earlier point in the code, if we can't handle lack of a trailing slash.

Since time is short and the rest of the PR looks good, I am going to restore the previous version of this line via a separate commit.

Reply:
I will move the normalization to IOUtils.getGenomicsDBAbsolutePath() in a future PR. Thanks.